### PR TITLE
Add k8s yaml file to support ACI

### PR DESCRIPTION
**Pull request by neelimamukiri**

### CODE REVIEW

File: install/k8s/README.md [hunk 1]
```diff
@@ -6,14 +6,31 @@ Install kubernetes 1.4 or higher using http://kubernetes.io/docs/getting-started
...
+    kubectl create secret generic aci.key --from-file=<path name of aci.key file> -n kube-system
+    ```
 * On the management node, run
 ```sh
 kubectl apply -f contiv.yaml
 ```
 * Get netctl from a Contiv release or local build. Contiv releases are available from https://github.com/contiv/netplugin/releases
+* Optional ACI only step:
+```sh
+netctl set global --fwd-mode routing
```
Comment (abhinandanpb) [inline install/k8s/README.md]: This is not a valid config in aci mode.

Comment (gaurav-dalvi) [inline install/k8s/README.md]: `netctl global set --fabric-mode aci --vlan-range 1150-1170`

vlan range must be configured in APIC before hand !

Comment (neelimamukiri) [inline install/k8s/README.md]: Thanks for checking. I've updated with the correct instruction for ACI/routing.

**Comment by abhinandanpb** [comment]
LGTM

**Pull request merged**